tests/conftest.py

```diff
@@ -74,7 +79,7 @@ def clipped_wflow_model():
 @pytest.fixture()
 def floodplain1d_testdata():
     data = xr.load_dataset(
-        join(TESTDATADIR, "floodplain_layers.nc"),
+        join(TESTDATADIR, SUBDIR, "floodplain_layers.nc"),
         lock=False,
         mode="r",
     )
```

Comment:
FYI: fixtures are scoped by module not by test by default in pytest so if you modify this in any way other tests will see that modification. Recently had to find out the hard way. You can add a scope argument to the fixture to change this. Not necessary but wanted to mention it anyway.

Reply:
Also something for a tests overhaul.